cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -196,82 +202,113 @@ public void init() throws ServletException{
                 .javalinServlet();
     }
 
-    private AccessManager buildAccessManager()
-    {
+    private AccessManager buildAccessManager() {
         return new CwmsAccessManager();
     }
 
 
 
 
-    protected void configureRoutes()
-    {
+    protected void configureRoutes() {
```

Comment:
Both are fine here, but FYI if it definitely makes more visual sense in a function to have long lines you can put a `@SuppressWarning("linelength")` (might not be exact name) to have checkstyle ignore it there.

I ran into a few of those when I was doing my big push.